README.md

```diff
@@ -117,30 +117,6 @@ npm run coverage
 * [Prettier](https://prettier.io/) (code formatter)
 
 
-## License
-
-[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```

Comment:
Maybe you can keep this flag and move it to the top of the readme

Reply:
fixed. I put back that badge.